source/includes/markdown/_13_1-scim.html.md

```diff
@@ -31,10 +57,24 @@ The accepted user attributes are:
 |name.givenName|string|Unsupported, use `formatted`|
 |name.familyName|string|Unsupported, use `formatted`|
 |name.formatted|string|The full name of the user.|
-|emails|complex|Email addresses for the user.|
+|emails|multi-valued complex|Email addresses for the user.|
 |emails.value|string|Email address for the user|
 |email.primary|string|Whether this email address is the preferred email address for this user. `true` may only appear once for this  attribute.|
 |active|boolean|Indicates whether the user's account is active in Asana.|
 |photo|string|URL for the user's profile photo.|
+|title|string|The user's title, such as "Vice President".|
+|preferredLanguage|string|The user's preferred language. Used for selecting the localized user interface.|
+|"urn:ietf:params:scim:<br>schemas:extension:enterprise:<br>2.0:User"|complex|The attribute definition.|
+|"urn:ietf:params:scim:<br>schemas:extension:enterprise:<br>2.0:User.department"|string|The department the user belongs to.|
+
+
+<br />
+The accepted group attributes are:
```

Comment:
Have you considered grouping the resources with attributes? For instance, User Resource followed by User attributes, and Group Resource followed by Group attributes?

Also, can we have a sub-sub-section for the attributes?

Reply:
I was thinking in having the attributes as part of it:

SCIM Endpoints
    User-related stuff
        Endpoints
        Attributes